openfe/tests/protocols/test_openmm_afe_slow.py

```diff
@@ -45,18 +45,20 @@ def test_openmm_run_engine(platform,
 
     # Run a really short calculation to check everything is going well
     s = openmm_afe.AbsoluteSolvationProtocol.default_settings()
-    s.alchemsampler_settings.n_repeats = 1
+    s.protocol_repeats = 1
```

Comment:
Looks like a lot of these slow tests didn't get fixed when the settings merge got made.